Style html elements

anirudh-v2 opened the pull request:
created latest stories aligned in a row, where each story has an image, caption and a headline .

Commit:
added 6 latest stories of today

Commit:
added six latest stories withnew images

Commit:
three rows created in a table

Commit:
initial commit of latest stories

Commit:
removed comment

Commit:
linked external stylesheet

Commit:
changed padding of column

manoj-v2 commented on styles.css:
```diff
@@ -0,0 +1,40 @@
+    margin-block-start: 0em;
+    margin-block-end: 0em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
```
Dont mix em and px. Use any one of them.

manoj-v2 replied:
you dont have to use inline-block etc.. you can just use margin and set the display to either block or inline

manoj-v2 commented on latest-stories.html:
```diff
@@ -3,63 +3,71 @@
+                Moeen urges England team to imporve off-field behaviour
+            </h4>
+        </li>
+        <li class="column" id="article-border-right">
```
id  are unique in a document.  https://www.w3schools.com/html/html_id.asp

manoj-v2 replied:
you can use pseudo class selector to disable border for the last li. https://www.w3schools.com/cssref/sel_last-child.asp

manoj-v2 commented on latest-stories.html:
```diff
@@ -3,63 +3,71 @@
+    </header>
+    <!-- first article -->
+    <ul>
+        <li class="column" id="article-border-right" style="padding: 0px 10px 0px 0px;">
```
No inline styles please.

Commit:
changed to common unit,px.
moved the style from inline to external style
removed article-border-right id from the elements

anirudh-v2 commented:
changed to a common unit, px.
moved the style from inline to external style.
removed article-border-right id from html elements.
